### PR TITLE
[GOBBLIN-1272] Fixing bug in loading config-store when the entry is empty

### PR DESCRIPTION
**Pull request by autumnust**
Dear Gobblin maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- https://issues.apache.org/jira/browse/GOBBLIN-1272

### Description
A tiny PR that handle the case when there's empty string specified in an entry of Config-Store properly.

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

File: gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/ConfigStoreUtils.java [hunk 1]
```diff
@@ -260,8 +262,11 @@ public static void setTopicsFromConfigStore(Properties properties, Set<String> b
   }
 
   public static List<String> getListOfValuesFromConfigStore(Config config, String keyValue) {
```
Comment (sv2000) [inline gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/ConfigStoreUtils.java:264]: Doesn't ConfigUtils#getStringList() achieve the same? If so, should we reuse the method defined there?

Comment (sv2000) [inline gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/ConfigStoreUtils.java:264]: Also - it seems like the more appropriate name of the 2nd argument should be key.

**Review by sv2000 — CHANGES_REQUESTED** [review]

File: gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/ConfigStoreUtils.java (hunk 1 above)
Comment (autumnust) [reply gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/ConfigStoreUtils.java:264]: That's a good catch, will address.

**Review by sv2000 — APPROVED** [review]
+1. LGTM.

**Pull request closed without merging**